tests/sentry/backup/test_models.py

```diff
@@ -217,6 +223,53 @@ def test_organization(self):
         self.create_organization(owner=user)
         return self.import_export_then_validate()
 
+    @targets_models(Incident, Organization, AlertRule)
+    def test_incident(self):
+        self.create_incident()
+        return self.import_export_then_validate()
+
+    @targets_models(IncidentActivity)
+    def test_incidentActivity(self):
+        IncidentActivity.objects.create(
+            incident=self.create_incident(),
+            type=1,
+            comment="hello",
+        )
+        return self.import_export_then_validate()
+
+    @targets_models(IncidentSnapshot, TimeSeriesSnapshot)
```

Comment:
We can probably just remove the `TimeSeriesSnapshot` test from `test_snapshots` down below in this case.